Make compatible with openjdk-9-jdk

When trying to run "mvn jetty:run" in a Vaadin archetype application based project on Ubuntu 16.04 with openjdk-9-jdk as the Java implementation, the command fails with "cannot access class sun.net.www.protocol.file.FileURLConnection".

Looks like [ClassPathExplorer](https://github.com/vaadin/maven-plugin/blob/4aa48062e9fc584385b299ad7525718415ad34a7/src/main/java/com/vaadin/integration/maven/wscdn/ClassPathExplorer.java#L39) tries to use a class sun.net.www.protocol.file.FileURLConnection that is not available in the openjdk-9-jdk distribution.

Also, Oracle says: "[The sun.* packages are not part of the supported, public interface.](http://www.oracle.com/technetwork/java/faq-sun-packages-142232.html)"

Full console output:
```
$ mvn --version
Apache Maven 3.3.9
Maven home: /usr/share/maven
Java version: 9-internal, vendor: Oracle Corporation
Java home: /usr/lib/jvm/java-9-openjdk-amd64
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "4.10.0-35-generic", arch: "amd64", family: "unix"
$ mvn jetty:run
[INFO] Scanning for projects...
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building vaadinfiddleprototype 1.0-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] >>> jetty-maven-plugin:9.3.9.v20160517:run (default-cli) > test-compile @ vaadinfiddleprototype >>>
[INFO] 
[INFO] --- vaadin-maven-plugin:8.1.3:update-theme (default) @ vaadinfiddleprototype ---
[INFO] Updating theme VAADIN/themes/vaadin-fiddle
[INFO] Widgetsets found from classpath:
[INFO] 	com.vaadin.contextmenu.WidgetSet in jar:file:/home/jonni/.m2/repository/com/vaadin/vaadin-context-menu/2.0.0/vaadin-context-menu-2.0.0.jar!/
[INFO] Addon styles found from classpath:
[INFO] 
[INFO] Search took 2ms
[INFO] Theme "VAADIN/themes/vaadin-fiddle" updated
[INFO] 
[INFO] --- vaadin-maven-plugin:8.1.3:update-widgetset (default) @ vaadinfiddleprototype ---
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 1.352 s
[INFO] Finished at: 2017-10-09T15:52:45+03:00
[INFO] Final Memory: 17M/58M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.vaadin:vaadin-maven-plugin:8.1.3:update-widgetset (default) on project vaadinfiddleprototype: Execution default of goal com.vaadin:vaadin-maven-plugin:8.1.3:update-widgetset failed: An API incompatibility was encountered while executing com.vaadin:vaadin-maven-plugin:8.1.3:update-widgetset: java.lang.IllegalAccessError: class com.vaadin.integration.maven.wscdn.ClassPathExplorer (in unnamed module @0x655a01d8) cannot access class sun.net.www.protocol.file.FileURLConnection (in module java.base) because module java.base does not export sun.net.www.protocol.file to unnamed module @0x655a01d8
[ERROR] -----------------------------------------------------
[ERROR] realm =    plugin>com.vaadin:vaadin-maven-plugin:8.1.3
[ERROR] strategy = org.codehaus.plexus.classworlds.strategy.SelfFirstStrategy
[ERROR] urls[0] = file:/home/jonni/.m2/repository/com/vaadin/vaadin-maven-plugin/8.1.3/vaadin-maven-plugin-8.1.3.jar
[ERROR] urls[1] = file:/home/jonni/.m2/repository/org/codehaus/plexus/plexus-interpolation/1.14/plexus-interpolation-1.14.jar
[ERROR] urls[2] = file:/home/jonni/.m2/repository/org/sonatype/sisu/sisu-inject-bean/2.3.0/sisu-inject-bean-2.3.0.jar
[ERROR] urls[3] = file:/home/jonni/.m2/repository/org/sonatype/sisu/sisu-guice/3.1.0/sisu-guice-3.1.0-no_aop.jar
[ERROR] urls[4] = file:/home/jonni/.m2/repository/org/sonatype/sisu/sisu-guava/0.9.9/sisu-guava-0.9.9.jar
[ERROR] urls[5] = file:/home/jonni/.m2/repository/org/codehaus/plexus/plexus-component-annotations/1.5.5/plexus-component-annotations-1.5.5.jar
[ERROR] urls[6] = file:/home/jonni/.m2/repository/org/sonatype/aether/aether-util/1.13.1/aether-util-1.13.1.jar
[ERROR] urls[7] = file:/home/jonni/.m2/repository/org/sonatype/plexus/plexus-sec-dispatcher/1.3/plexus-sec-dispatcher-1.3.jar
[ERROR] urls[8] = file:/home/jonni/.m2/repository/org/sonatype/plexus/plexus-cipher/1.4/plexus-cipher-1.4.jar
[ERROR] urls[9] = file:/home/jonni/.m2/repository/org/apache/maven/reporting/maven-reporting-api/3.0/maven-reporting-api-3.0.jar
[ERROR] urls[10] = file:/home/jonni/.m2/repository/org/apache/maven/reporting/maven-reporting-impl/2.1/maven-reporting-impl-2.1.jar
[ERROR] urls[11] = file:/home/jonni/.m2/repository/org/apache/maven/doxia/doxia-core/1.1.2/doxia-core-1.1.2.jar
[ERROR] urls[12] = file:/home/jonni/.m2/repository/commons-httpclient/commons-httpclient/3.1/commons-httpclient-3.1.jar
[ERROR] urls[13] = file:/home/jonni/.m2/repository/commons-validator/commons-validator/1.2.0/commons-validator-1.2.0.jar
[ERROR] urls[14] = file:/home/jonni/.m2/repository/commons-beanutils/commons-beanutils/1.7.0/commons-beanutils-1.7.0.jar
[ERROR] urls[15] = file:/home/jonni/.m2/repository/commons-digester/commons-digester/1.6/commons-digester-1.6.jar
[ERROR] urls[16] = file:/home/jonni/.m2/repository/oro/oro/2.0.8/oro-2.0.8.jar
[ERROR] urls[17] = file:/home/jonni/.m2/repository/org/apache/xbean/xbean-reflect/3.4/xbean-reflect-3.4.jar
[ERROR] urls[18] = file:/home/jonni/.m2/repository/log4j/log4j/1.2.12/log4j-1.2.12.jar
[ERROR] urls[19] = file:/home/jonni/.m2/repository/commons-logging/commons-logging-api/1.1/commons-logging-api-1.1.jar
[ERROR] urls[20] = file:/home/jonni/.m2/repository/com/google/collections/google-collections/1.0/google-collections-1.0.jar
[ERROR] urls[21] = file:/home/jonni/.m2/repository/org/apache/maven/doxia/doxia-sink-api/1.6/doxia-sink-api-1.6.jar
[ERROR] urls[22] = file:/home/jonni/.m2/repository/org/apache/maven/doxia/doxia-logging-api/1.6/doxia-logging-api-1.6.jar
[ERROR] urls[23] = file:/home/jonni/.m2/repository/org/apache/maven/doxia/doxia-site-renderer/1.6/doxia-site-renderer-1.6.jar
[ERROR] urls[24] = file:/home/jonni/.m2/repository/org/apache/maven/doxia/doxia-decoration-model/1.6/doxia-decoration-model-1.6.jar
[ERROR] urls[25] = file:/home/jonni/.m2/repository/org/apache/maven/doxia/doxia-module-xhtml/1.6/doxia-module-xhtml-1.6.jar
[ERROR] urls[26] = file:/home/jonni/.m2/repository/org/apache/maven/doxia/doxia-module-fml/1.6/doxia-module-fml-1.6.jar
[ERROR] urls[27] = file:/home/jonni/.m2/repository/org/codehaus/plexus/plexus-i18n/1.0-beta-7/plexus-i18n-1.0-beta-7.jar
[ERROR] urls[28] = file:/home/jonni/.m2/repository/org/codehaus/plexus/plexus-velocity/1.1.7/plexus-velocity-1.1.7.jar
[ERROR] urls[29] = file:/home/jonni/.m2/repository/org/apache/velocity/velocity/1.5/velocity-1.5.jar
[ERROR] urls[30] = file:/home/jonni/.m2/repository/org/apache/velocity/velocity-tools/2.0/velocity-tools-2.0.jar
[ERROR] urls[31] = file:/home/jonni/.m2/repository/commons-chain/commons-chain/1.1/commons-chain-1.1.jar
[ERROR] urls[32] = file:/home/jonni/.m2/repository/sslext/sslext/1.2-0/sslext-1.2-0.jar
[ERROR] urls[33] = file:/home/jonni/.m2/repository/org/apache/struts/struts-core/1.3.8/struts-core-1.3.8.jar
[ERROR] urls[34] = file:/home/jonni/.m2/repository/antlr/antlr/2.7.2/antlr-2.7.2.jar
[ERROR] urls[35] = file:/home/jonni/.m2/repository/org/apache/struts/struts-taglib/1.3.8/struts-taglib-1.3.8.jar
[ERROR] urls[36] = file:/home/jonni/.m2/repository/org/apache/struts/struts-tiles/1.3.8/struts-tiles-1.3.8.jar
[ERROR] urls[37] = file:/home/jonni/.m2/repository/commons-collections/commons-collections/3.2.1/commons-collections-3.2.1.jar
[ERROR] urls[38] = file:/home/jonni/.m2/repository/org/springframework/spring-core/3.0.5.RELEASE/spring-core-3.0.5.RELEASE.jar
[ERROR] urls[39] = file:/home/jonni/.m2/repository/org/springframework/spring-asm/3.0.5.RELEASE/spring-asm-3.0.5.RELEASE.jar
[ERROR] urls[40] = file:/home/jonni/.m2/repository/commons-logging/commons-logging/1.1.1/commons-logging-1.1.1.jar
[ERROR] urls[41] = file:/home/jonni/.m2/repository/org/codehaus/plexus/plexus-utils/3.0.18/plexus-utils-3.0.18.jar
[ERROR] urls[42] = file:/home/jonni/.m2/repository/com/thoughtworks/qdox/qdox/1.12.1/qdox-1.12.1.jar
[ERROR] urls[43] = file:/home/jonni/.m2/repository/org/freemarker/freemarker/2.3.20/freemarker-2.3.20.jar
[ERROR] urls[44] = file:/home/jonni/.m2/repository/com/vaadin/wscdn/wscdn-client/0.9.2/wscdn-client-0.9.2.jar
[ERROR] urls[45] = file:/home/jonni/.m2/repository/commons-codec/commons-codec/1.10/commons-codec-1.10.jar
[ERROR] urls[46] = file:/home/jonni/.m2/repository/org/jboss/resteasy/resteasy-client/3.0.10.Final/resteasy-client-3.0.10.Final.jar
[ERROR] urls[47] = file:/home/jonni/.m2/repository/org/jboss/resteasy/resteasy-jaxrs/3.0.10.Final/resteasy-jaxrs-3.0.10.Final.jar
[ERROR] urls[48] = file:/home/jonni/.m2/repository/org/jboss/resteasy/jaxrs-api/3.0.10.Final/jaxrs-api-3.0.10.Final.jar
[ERROR] urls[49] = file:/home/jonni/.m2/repository/org/jboss/spec/javax/annotation/jboss-annotations-api_1.1_spec/1.0.1.Final/jboss-annotations-api_1.1_spec-1.0.1.Final.jar
[ERROR] urls[50] = file:/home/jonni/.m2/repository/javax/activation/activation/1.1/activation-1.1.jar
[ERROR] urls[51] = file:/home/jonni/.m2/repository/org/apache/httpcomponents/httpclient/4.2.6/httpclient-4.2.6.jar
[ERROR] urls[52] = file:/home/jonni/.m2/repository/org/apache/httpcomponents/httpcore/4.2.5/httpcore-4.2.5.jar
[ERROR] urls[53] = file:/home/jonni/.m2/repository/net/jcip/jcip-annotations/1.0/jcip-annotations-1.0.jar
[ERROR] urls[54] = file:/home/jonni/.m2/repository/org/jboss/resteasy/resteasy-jaxb-provider/3.0.10.Final/resteasy-jaxb-provider-3.0.10.Final.jar
[ERROR] urls[55] = file:/home/jonni/.m2/repository/com/sun/xml/bind/jaxb-impl/2.2.7/jaxb-impl-2.2.7.jar
[ERROR] urls[56] = file:/home/jonni/.m2/repository/com/sun/xml/bind/jaxb-core/2.2.7/jaxb-core-2.2.7.jar
[ERROR] urls[57] = file:/home/jonni/.m2/repository/javax/xml/bind/jaxb-api/2.2.7/jaxb-api-2.2.7.jar
[ERROR] urls[58] = file:/home/jonni/.m2/repository/com/sun/istack/istack-commons-runtime/2.16/istack-commons-runtime-2.16.jar
[ERROR] urls[59] = file:/home/jonni/.m2/repository/com/sun/xml/fastinfoset/FastInfoset/1.2.12/FastInfoset-1.2.12.jar
[ERROR] urls[60] = file:/home/jonni/.m2/repository/javax/xml/bind/jsr173_api/1.0/jsr173_api-1.0.jar
[ERROR] urls[61] = file:/home/jonni/.m2/repository/org/jboss/resteasy/resteasy-jackson2-provider/3.0.10.Final/resteasy-jackson2-provider-3.0.10.Final.jar
[ERROR] urls[62] = file:/home/jonni/.m2/repository/com/fasterxml/jackson/core/jackson-core/2.4.1/jackson-core-2.4.1.jar
[ERROR] urls[63] = file:/home/jonni/.m2/repository/com/fasterxml/jackson/core/jackson-databind/2.4.1/jackson-databind-2.4.1.jar
[ERROR] urls[64] = file:/home/jonni/.m2/repository/com/fasterxml/jackson/core/jackson-annotations/2.4.1/jackson-annotations-2.4.1.jar
[ERROR] urls[65] = file:/home/jonni/.m2/repository/com/fasterxml/jackson/jaxrs/jackson-jaxrs-json-provider/2.4.1/jackson-jaxrs-json-provider-2.4.1.jar
[ERROR] urls[66] = file:/home/jonni/.m2/repository/com/fasterxml/jackson/jaxrs/jackson-jaxrs-base/2.4.1/jackson-jaxrs-base-2.4.1.jar
[ERROR] urls[67] = file:/home/jonni/.m2/repository/com/fasterxml/jackson/module/jackson-module-jaxb-annotations/2.4.1/jackson-module-jaxb-annotations-2.4.1.jar
[ERROR] urls[68] = file:/home/jonni/.m2/repository/org/codehaus/plexus/plexus-archiver/2.6.3/plexus-archiver-2.6.3.jar
[ERROR] urls[69] = file:/home/jonni/.m2/repository/org/codehaus/plexus/plexus-io/2.1.3/plexus-io-2.1.3.jar
[ERROR] urls[70] = file:/home/jonni/.m2/repository/org/apache/commons/commons-compress/1.8.1/commons-compress-1.8.1.jar
[ERROR] urls[71] = file:/home/jonni/.m2/repository/org/apache/maven/maven-archiver/2.5/maven-archiver-2.5.jar
[ERROR] urls[72] = file:/home/jonni/.m2/repository/junit/junit/4.11/junit-4.11.jar
[ERROR] urls[73] = file:/home/jonni/.m2/repository/org/hamcrest/hamcrest-core/1.3/hamcrest-core-1.3.jar
[ERROR] urls[74] = file:/home/jonni/.m2/repository/org/apache/maven/surefire/surefire-booter/2.4.3/surefire-booter-2.4.3.jar
[ERROR] urls[75] = file:/home/jonni/.m2/repository/org/apache/maven/surefire/surefire-api/2.4.3/surefire-api-2.4.3.jar
[ERROR] urls[76] = file:/home/jonni/.m2/repository/commons-io/commons-io/2.4/commons-io-2.4.jar
[ERROR] urls[77] = file:/home/jonni/.m2/repository/commons-lang/commons-lang/2.6/commons-lang-2.6.jar
[ERROR] urls[78] = file:/home/jonni/.m2/repository/org/jdom/jdom/1.1.3/jdom-1.1.3.jar
[ERROR] urls[79] = file:/home/jonni/.m2/repository/jaxen/jaxen/1.1.1/jaxen-1.1.1.jar
[ERROR] urls[80] = file:/home/jonni/.m2/repository/dom4j/dom4j/1.6.1/dom4j-1.6.1.jar
[ERROR] urls[81] = file:/home/jonni/.m2/repository/jdom/jdom/1.0/jdom-1.0.jar
[ERROR] urls[82] = file:/home/jonni/.m2/repository/xom/xom/1.0/xom-1.0.jar
[ERROR] urls[83] = file:/home/jonni/.m2/repository/xerces/xmlParserAPIs/2.6.2/xmlParserAPIs-2.6.2.jar
[ERROR] urls[84] = file:/home/jonni/.m2/repository/xalan/xalan/2.6.0/xalan-2.6.0.jar
[ERROR] urls[85] = file:/home/jonni/.m2/repository/com/ibm/icu/icu4j/3.4.4/icu4j-3.4.4.jar
[ERROR] urls[86] = file:/home/jonni/.m2/repository/xerces/xercesImpl/2.10.0/xercesImpl-2.10.0.jar
[ERROR] urls[87] = file:/home/jonni/.m2/repository/org/codehaus/plexus/plexus-compiler-api/2.3/plexus-compiler-api-2.3.jar
[ERROR] urls[88] = file:/home/jonni/.m2/repository/org/sonatype/plexus/plexus-build-api/0.0.7/plexus-build-api-0.0.7.jar
[ERROR] urls[89] = file:/home/jonni/.m2/repository/org/codehaus/plexus/plexus-compiler-javac/2.3/plexus-compiler-javac-2.3.jar
[ERROR] urls[90] = file:/home/jonni/.m2/repository/xml-apis/xml-apis/1.4.01/xml-apis-1.4.01.jar
[ERROR] Number of foreign imports: 1
[ERROR] import: Entry[import  from realm ClassRealm[maven.api, parent: null]]
[ERROR] 
[ERROR] -----------------------------------------------------
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginContainerException
```

--- a/pom.xml
+++ b/pom.xml
@@ -109,7 +109,7 @@
     <!-- TODO this would also be used in an archetype, currently broken -->
     <!-- This property is used in a filtered resources to check the version compatibility -->
     <vaadin.version>${project.version}</vaadin.version>
-    <gwt.version>2.8.0</gwt.version>
+    <gwt.version>2.8.2</gwt.version>
     <!--  apt filtering doesn't support dot -->
     <vaadinVersion>${vaadin.version}</vaadinVersion>
     <gwtVersion>${gwt.version}</gwtVersion>

--- a/src/main/java/com/vaadin/integration/maven/wscdn/ClassPathExplorer.java
+++ b/src/main/java/com/vaadin/integration/maven/wscdn/ClassPathExplorer.java
@@ -38,8 +38,6 @@ import java.util.jar.Manifest;
 
 import com.vaadin.integration.maven.wscdn.CvalChecker.UnreachableCvalServerException;
 
-import sun.net.www.protocol.file.FileURLConnection;
-
 /**
  * Utility class to collect widgetset related information from classpath.
  * Utility will seek all directories from classpaths, and jar files having
@@ -238,86 +236,84 @@ public class ClassPathExplorer {
                 // and files in jar
 
                 URLConnection openConnection = location.openConnection();
-                if (openConnection instanceof JarURLConnection || openConnection instanceof FileURLConnection) {
 
-                    JarFile jarFile;
-                    if (openConnection instanceof JarURLConnection) {
-                        JarURLConnection conn = (JarURLConnection) openConnection;
-                        jarFile = conn.getJarFile();
-                    } else {
-                        jarFile = new JarFile(location.getFile());
-                    }
+                JarFile jarFile;
+                if (openConnection instanceof JarURLConnection) {
+                    JarURLConnection conn = (JarURLConnection) openConnection;
+                    jarFile = conn.getJarFile();
+                } else {
+                    jarFile = new JarFile(location.getFile());
+                }
 
-                    Manifest manifest = jarFile.getManifest();
-                    if (manifest == null) {
-                        // No manifest so this is not a Vaadin Add-on
-                        return;
-                    }
+                Manifest manifest = jarFile.getManifest();
+                if (manifest == null) {
+                    // No manifest so this is not a Vaadin Add-on
+                    return;
+                }
 
-                    // Check for widgetset attribute
-                    String value = manifest.getMainAttributes().getValue(
-                            "Vaadin-Widgetsets");
-                    if (value != null) {
-                        String[] widgetsetNames = value.split(",");
-                        for (int i = 0; i < widgetsetNames.length; i++) {
-                            String widgetsetname = widgetsetNames[i].trim();
-                            if (!widgetsetname.equals("")) {
-                                widgetsets.put(widgetsetname, location);
-                            }
-                        }
-
-                        Attributes attribs = manifest.getMainAttributes();
-                        String license = attribs.getValue(VAADIN_ADDON_LICENSE);
-                        String name = attribs.getValue(VAADIN_ADDON_NAME);
-                        String vers = attribs.getValue(VAADIN_ADDON_VERSION) == null ? ""
-                                : attribs.getValue(VAADIN_ADDON_VERSION);
-                        String title = attribs.getValue(VAADIN_ADDON_TITLE) == null ? name
-                                : attribs.getValue(VAADIN_ADDON_TITLE);
-
-                        String awidgetsets = attribs
-                                .getValue(VAADIN_ADDON_WIDGETSET) == null ? name
-                                        : attribs.getValue(
-                                                VAADIN_ADDON_WIDGETSET);
-
-                        if (name != null && license != null) {
-                            if (VAADIN_AGPL.equals(license)) {
-                                // For agpl version, we don't care
-                            } else if (VAADIN_CVAL.equals(license)) {
-                                // We only check cval licensed products
-                                CvalInfo info;
-
-                                try {
-                                    info = cvalChecker.validateProduct(name,
-                                            vers,
-                                            title);
-                                    printValidLicense(info, title, vers);
-                                } catch (UnreachableCvalServerException e) {
-                                    info = new CvalInfo();
-                                    final Product product = new Product();
-                                    product.setName(name);
-                                    info.setProduct(product);
-                                    printServerUnreachable(title, vers);
-                                }
-//                                for (String w : awidgetsets.split("[, ]+")) {
-//                                    ret.add(new CValUiInfo(title, String
-//                                            .valueOf(computeMajorVersion(vers)),
-//                                            w,
-//                                            info.getType()));
-//                                }
-                            }
+                // Check for widgetset attribute
+                String value = manifest.getMainAttributes().getValue(
+                        "Vaadin-Widgetsets");
+                if (value != null) {
+                    String[] widgetsetNames = value.split(",");
+                    for (int i = 0; i < widgetsetNames.length; i++) {
+                        String widgetsetname = widgetsetNames[i].trim();
+                        if (!widgetsetname.equals("")) {
+                            widgetsets.put(widgetsetname, location);
                         }
                     }
 
-                    // Check for theme attribute
-                    value = manifest.getMainAttributes().getValue(
-                            "Vaadin-Stylesheets");
-                    if (value != null) {
-                        String[] stylesheets = value.split(",");
-                        for (int i = 0; i < stylesheets.length; i++) {
-                            String stylesheet = stylesheets[i].trim();
-                            if (!stylesheet.equals("")) {
-                                addonStyles.put(stylesheet, location);
+                    Attributes attribs = manifest.getMainAttributes();
+                    String license = attribs.getValue(VAADIN_ADDON_LICENSE);
+                    String name = attribs.getValue(VAADIN_ADDON_NAME);
+                    String vers = attribs.getValue(VAADIN_ADDON_VERSION) == null ? ""
+                            : attribs.getValue(VAADIN_ADDON_VERSION);
+                    String title = attribs.getValue(VAADIN_ADDON_TITLE) == null ? name
+                            : attribs.getValue(VAADIN_ADDON_TITLE);
+
+                    String awidgetsets = attribs
+                            .getValue(VAADIN_ADDON_WIDGETSET) == null ? name
+                                    : attribs.getValue(
+                                            VAADIN_ADDON_WIDGETSET);
+
+                    if (name != null && license != null) {
+                        if (VAADIN_AGPL.equals(license)) {
+                            // For agpl version, we don't care
+                        } else if (VAADIN_CVAL.equals(license)) {
+                            // We only check cval licensed products
+                            CvalInfo info;
+
+                            try {
+                                info = cvalChecker.validateProduct(name,
+                                        vers,
+                                        title);
+                                printValidLicense(info, title, vers);
+                            } catch (UnreachableCvalServerException e) {
+                                info = new CvalInfo();
+                                final Product product = new Product();
+                                product.setName(name);
+                                info.setProduct(product);
+                                printServerUnreachable(title, vers);
                             }
+//                          for (String w : awidgetsets.split("[, ]+")) {
+//                              ret.add(new CValUiInfo(title, String
+//                                      .valueOf(computeMajorVersion(vers)),
+//                                      w,
+//                                      info.getType()));
+//                          }
+                        }
+                    }
+                }
+
+                // Check for theme attribute
+                value = manifest.getMainAttributes().getValue(
+                        "Vaadin-Stylesheets");
+                if (value != null) {
+                    String[] stylesheets = value.split(",");
+                    for (int i = 0; i < stylesheets.length; i++) {
+                        String stylesheet = stylesheets[i].trim();
+                        if (!stylesheet.equals("")) {
+                            addonStyles.put(stylesheet, location);
                         }
                     }
                 }

